core/grid/AjaxActionColumn.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace core\grid;
+
+use Yii;
+use yii\helpers\Html;
+
+class AjaxActionColumn extends ActionColumn
```

Comment:
we need this class to add ajax css classes to buttons, right?

Reply:
this class is used in ActionColumns of grid view to sending ajax request to controller and triggering sliding form to open and close